dll/win32/newdev/newdev.c

```diff
@@ -389,9 +402,8 @@ SearchDriverRecursive(
             {
                 wcscpy(LastDirPath, DirPath);
 
-                if (wcslen(DirPath) > MAX_PATH)
-                    /* Path is too long to be searched */
-                    continue;
+                /* if wcslen(DirPath) >= MAX_PATH here it's too late, program has already crashed */
```

Comment:
Then, shouldn't this check be moved (earlier), where it would not be useless?

Reply:
I don't understand. I'm agree that the test is useless, it is why I removed it and moved earlier. I have replaced it by an assert because now, it should never happens.